generators/plopGenerateModule.js

```diff
@@ -0,0 +1,28 @@
+﻿module.exports = function (plop) {
+  plop.setGenerator('component', {
+    description: 'Criação de componente',
```

Comment:
```suggestion
    description: 'Criação de módulo',
```
Acredito que vale essa alteração em outros pontos deste arquivo.